scripts/import_pjsip_config.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+# Copyright 2020 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0+
+
+import sys
+import argparse
+import configparser
+import os
+
+from requests import HTTPError
+from wazo_auth_client import Client as AuthClient
+from wazo_confd_client import Client as ConfdClient
+from xivo.chain_map import ChainMap
+from xivo.config_helper import read_config_file_hierarchy, parse_config_file
+
+_DEFAULT_CONFIG = {
+    'config_file': '/etc/wazo-upgrade/config.yml',
+    'auth': {
+        'key_file': '/var/lib/wazo-auth-keys/wazo-upgrade-key.yml'
+    }
+}
+IGNORED_OPTIONS = [
+    'endpoint',
+]
+
+
+def load_config():
+    file_config = read_config_file_hierarchy(_DEFAULT_CONFIG)
+    key_config = _load_key_file(ChainMap(file_config, _DEFAULT_CONFIG))
+    return ChainMap(key_config, file_config, _DEFAULT_CONFIG)
+
+
+def _load_key_file(config):
+    key_file = parse_config_file(config['auth']['key_file'])
+    return {'auth': {'username': key_file['service_id'],
+                     'password': key_file['service_key']}}
+
+
+
+argument_parser = argparse.ArgumentParser(
+    'Import an Asterisk pjsip configuration file',
+)
+argument_parser.add_argument('filename', help='The file to import')
+
+
+def migrate_section(confd_client, block_name, options):
+    # if the names is reg_<endpoint_name>@host we are in a registration section
+    if 'reg_' and '@' in block_name:
```

Comment:
`'reg_'` is always evaluate to `True` :thinking: 
```suggestion
    if block_name and block_name.startswith('reg_') and '@' in block_name:
```